docs/page-objects/additional-requests.rst

```diff
@@ -773,3 +773,53 @@ Here's an example:
 From the example above, we're now checking the list of responses to see if any
 exceptions are included in it. If so, we're simply logging it down and ignoring
 it. In this way, perfectly good responses can still be processed through.
+
+
+.. _retries-additional-requests:
+
+Retrying Additional Requests
+============================
+
+When the bad response data comes from :ref:`additional requests
+<additional-requests>`, you must handle retries on your own.
+
+The page object code is responsible for retrying additional requests until good
+response data is received, or until some maximum number of retries is exceeded.
+
+It is up to you to decide what the maximum number of retries should be for a
+given additional request, based on your experience with the target website.
+
+It is also up to you to decide how to implement retries of additional requests.
+
+One option would be tenacity_. For example, to try an additional request 3
+times before giving up:
+
+.. _tenacity: https://tenacity.readthedocs.io/en/latest/index.html
+
+.. code-block:: python
+
+    import attrs
+    from tenacity import retry, stop_after_attempt
+    from web_poet import HttpClient, WebPage
+
+    @attrs.define
+    class MyPage(WebPage):
+        http: HttpClient
+
+        @retry(stop=stop_after_attempt(3))
```

Comment:
We should encapsulate 3rd party lib functionality into wrappers in case we need to migrate the functionality to other library on the future.

Reply:
How people implement retries for this scenario is not part of the web-poet spec, this is just an example. People are free to use something other than tenacity.